cmd/login.go

```diff
@@ -8,6 +8,19 @@ import (
 	"github.com/urfave/cli"
 )
 
+func AutoRefreshVaultToken(app *cli.App) {
+	login := config.Get().LoadLogin()
+	if login.Auth != "vault" || login.VaultUrl == "" || login.VaultToken == "" {
+		return
+	}
+
+	args := []string{
+		app.Name, "login", "--auth", login.Auth, "--vault-token",
+		login.VaultToken, "--vault-url", login.VaultUrl, login.Endpoint}
+
+	app.Run(args)
+}
+
```

Comment:
Don't use the CLI interface for this. If the CLI has to change this has to change as well. Either move the `vaultLogin` method to utils and call it from `client` or perform the request in the `client` package.